eo-maven-plugin/src/test/java/org/eolang/maven/ParseMojoTest.java

```diff
@@ -84,22 +84,22 @@ void failsOnTimeout(@TempDir final Path temp) {
     @Test
     @ExtendWith(OnlineCondition.class)
     void parsesWithCache(@TempDir final Path temp) throws Exception {
-        final FakeMaven maven = new FakeMaven(temp);
         final Path cache = temp.resolve("cache");
+        final FakeMaven maven = new FakeMaven(temp)
+            .withProgram("invalid content")
+            .with("cache", cache);
         final String expected = new UncheckedText(
             new TextOf(new ResourceOf("org/eolang/maven/main.xmir"))
         ).asString();
-        final String hash = new ChNarrow(new ChRemote("0.25.0")).value();
+        final CommitHash hash = new ChNarrow(new ChRemote("0.25.0"));
         new FtCached(
-            new CacheVersion(FakeMaven.pluginVersion(), hash),
+            new CacheVersion(FakeMaven.pluginVersion(), hash.value()),
             cache.resolve(ParseMojo.PARSED),
             new FtDefault(maven.targetPath())
         ).save("foo.x.main", "xmir", () -> expected);
         MatcherAssert.assertThat(
```

Comment:
@volodya-lombrozo what about adding message to the test? Or it's out of the scope of your task?

Reply:
@maxonfjvipon Actually, it is. Moreover, we are going to add messages for all the assertions when the new version of [jtcop](https://github.com/volodya-lombrozo/jtcop) will be released.